docs/getting-started.md

```diff
@@ -1,67 +1,96 @@
+Start by adding the following to your module's `build.gradle`/`build.gradle.kts` file:
 
-Add the following to your `build.gradle`:
+=== "Groovy"
 
-```groovy
-android {
-  testOptions {
-    unitTests {
-      includeAndroidResources = true
+    ```groovy
+    android {
+      testOptions {
+        unitTests {
+          includeAndroidResources = true
+        }
+      }
     }
-  }
-}
 
-dependencies {
-  testImplementation 'junit:junit:4.13.2'
-  testImplementation 'org.robolectric:robolectric:{{ robolectric.version.current_patched }}'
-}
-```
+    dependencies {
+      testImplementation 'junit:junit:4.13.2'
+      testImplementation 'org.robolectric:robolectric:{{ robolectric.version.current_patched }}'
+    }
+    ```
 
-Add this line to your `gradle.properties` (no longer necessary with Android Studio 3.3+):
+=== "Kotlin DSL"
 
-```properties
-android.enableUnitTestBinaryResources=true
-```
+    ```kotlin
+    android {
+      testOptions {
+        unitTests {
+          isIncludeAndroidResources = true
+        }
+      }
+    }
+
+    dependencies {
+      testImplementation("junit:junit:4.13.2")
+      testImplementation("org.robolectric:robolectric:{{ robolectric.version.current_patched }}")
+    }
+    ```
 
-Annotate your test with the Robolectric test runner:
+Then, mark your test to run with `RobolectricTestRunner`:
 
-```java
-@RunWith(RobolectricTestRunner.class)
-public class SandwichTest {
-}
-```
+=== "Java"
+
+    ```java
+    import org.robolectric.RobolectricTestRunner;
+
+    @RunWith(RobolectricTestRunner.class)
+    public class SandwichTest {
+    }
+    ```
+
+=== "Kotlin"
+
+    ```kotlin
+    import org.robolectric.RobolectricTestRunner
+
+    @RunWith(RobolectricTestRunner::class)
+    public class SandwichTest
+    ```
 
 ## Building with Bazel
 
-Robolectric works with [Bazel](https://bazel.build) 0.10.0 or higher. Bazel integrates with Robolectric through the `android_local_test` rule. The Robolectric Java test code is the same for a Bazel project as a new Gradle project.
+Robolectric works with [Bazel](https://bazel.build) 0.10.0 or higher. Bazel integrates with Robolectric through the `android_local_test` rule. The Robolectric Java/Kotlin test code is the same for a Bazel project as for a Gradle project (see section above).
 
-Robolectric needs to be added as a dependency to your Bazel project with [`rules_jvm_external`](https://github.com/bazelbuild/rules_jvm_external). Add the following to your WORKSPACE file:
+Robolectric needs to be added as a dependency to your Bazel project with [`rules_jvm_external`](https://github.com/bazelbuild/rules_jvm_external). Add the following to your `WORKSPACE` file:
 
 ```python
 http_archive(
     name = "robolectric",
-    urls = ["https://github.com/robolectric/robolectric-bazel/archive/4.7.3.tar.gz"],
-    strip_prefix = "robolectric-bazel-4.7.3",
+    sha256 = "f7b8e08f246f29f26fddd97b7ab5dfa01aaa6170ccc24b9b6a21931bde01ad6f",
```

Comment:
Good point!
We can either, as you suggested, leave that section for a specific version, knowing that it will end up outdated, but add a link to [`robolectric-bazel` releases](https://github.com/robolectric/robolectric-bazel/releases) page.
Or, and maybe it's too extreme, I don't know how much Bazel is used outside Google, drop that section completely, and add an entry on `Additional resources` pointing to [`robolectric-bazel` latest release](https://github.com/robolectric/robolectric-bazel/releases/latest) for instructions.
What do you think?

Reply:
I've pushed some new changes in https://github.com/robolectric/robolectric.github.io/compare/c05940dd69c7c6932a435c9bc5f6de8798bd2842..fe523f9b9519b8dce8087003cf49f4da474b848e

- `robolectric-bazel` is hardcoded to use 4.12.2.
- I gave more visibility to the fact that the instructions are for specific versions of `robolectric-bazel` and `rules_jvm_external`, and to the link to the latest instructions.
- I've added the `github-callouts` extension, so now we can use [GitHub's alerts format](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) in Markdown